### ISSUE TITLE
problem with code mode and firefox

### ISSUE THREAD

**Issue by johbuch** [mautic/mautic#7340]
[//]: # ( Invisible comment: 
IIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIII
Before you create the issue:
IIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIII
Search for similar report among other reported issues.
Learn how to troubleshoot at https://www.mautic.org/docs/en/tips/troubleshooting.html
Use drag&drop to attach images or other files )

## Bug Description
There is a problem with code mode for email and firefox.
if you copy an HTML code in the code mode generator, then when applying it will work if your browser is Chrome but if it is firefox it won't work.

| Q   | A
| --- | ---
| Mautic version | 2.15.0 and 2.15.1
| PHP version | 
| Browser | 

### Steps to reproduce
1. Use firefox
2. Create a new email (add object and name)
3. Choose code mode
4. Copy and paste this html 
```
<!DOCTYPE html>
<html>
<body>
<h1>Test Heading</h1>
<p>Test paragraph.</p>
</body>
</html>
```
5. Click on apply
6. See that it still tries to load
![image](https://user-images.githubusercontent.com/31535432/54704796-c0203280-4b3b-11e9-9473-a9f62ff7710d.png)
And check the console inspector, you will see an error
<img width="824" alt="Capture d’écran 2019-03-20 à 18 09 51" src="https://user-images.githubusercontent.com/31535432/54704845-d62df300-4b3b-11e9-8ba1-6b0e5065914f.png">
7. Repeat this step with google chrome and you won't get a problem.
 
### Log errors

[//]: # ( Invisible comment:
Please check for related errors in the latest log file in [mautic root]/app/log/ and/or the web server's logs and post them here. Be sure to remove sensitive information if applicable. )

**Comment by vicampuzano** [comment]
I get this error (both on Chrome and Firefox) with html code generated with tools like beefree (dot) io or stripo (dot) email

**Comment by kuzmany** [comment]
Let's test https://github.com/mautic/mautic/pull/7346
On production server require regenerate assets `php app/console mautic:assets:generate`

**Comment by npracht** [comment]
Merged in 2.15.2
